src/GraphQL/Execution/SerialExecutionStrategy.cs

```diff
@@ -10,6 +10,7 @@ public class SerialExecutionStrategy : ExecutionStrategy
         // frequently reused objects
         private Stack<ExecutionNode> _reusableNodes;
         private Queue<ExecutionNode> _reusableDataLoaderNodes;
+        private Stack<ExecutionNode> _reusableAddlNodes;
```

Comment:
addl ?

Reply:
Worth to add comments: _reusableNodes are for ....  _reusableDataLoaderNodes are for ...  _reusableAddlNodes are for...

Reply:
> addl ?

Add'l is short for 'additional'